draft-ietf-ice-rfc5245bis.xml

```diff
@@ -415,16 +417,15 @@ stream that require their own set of candidates, e.g., RTP and RTCP).
 <section title="Nominating Candidate Pairs And Concluding ICE">
 <t>
 ICE assigns one of the ICE agents in the role of the
-controlling agent, and the other of the controlled agent. For each
-component of a data stream, the controlling agent nominates a candidate 
-pair from the valid candidate pairs to be used for data. The exact timing
-of the nomination is based on local policy.
+controlling agent, and the other of the controlled agent. For each 
+component of a data stream, the controlling agent nominates
+a valid pair (from the valid list) to be used for data.
```

Comment:
I would have kept the comment about timing

Reply:
I didn't intend to remove the comment. I'll put it back.